volatility3/framework/plugins/windows/psxview.py

```diff
@@ -0,0 +1,158 @@
+import datetime
+from typing import List, Iterable, Set, Dict
+from volatility3.framework import renderers, interfaces
+from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces import plugins
+from volatility3.plugins.windows import pslist, psscan
+
+class PsXview(plugins.PluginInterface):
+    """Find hidden processes with various process listings"""
+    _required_framework_version = (2, 0, 0)
+
+    @classmethod
+    def get_requirements(cls) -> List[interfaces.configuration.RequirementInterface]:
+        return [
+            requirements.TranslationLayerRequirement(name='primary',
+                            description='Memory layer for the kernel',
+                            architectures=["Intel32", "Intel64"]),
+            requirements.SymbolTableRequirement(name="nt_symbols",
+                            description="Windows kernel symbols"),
+        ]
+
+    @classmethod
+    def check_pslist(cls, context: interfaces.context.ContextInterface, layer_name: str, symbol_table: str) -> Set[int]:
+        """Enumerate processes from PsActiveProcessHead"""
+        return {proc.UniqueProcessId for proc in pslist.PsList.list_processes(context, layer_name, symbol_table)}
+
+    @classmethod
+    def check_psscan(cls, context: interfaces.context.ContextInterface, layer_name: str, symbol_table: str) -> Iterable[interfaces.objects.ObjectInterface]:
+        """Enumerate processes with pool tag scanning"""
+        return psscan.PsScan.scan_processes(context, layer_name, symbol_table)
```

Comment:
When using other plugins, you'd need to have a `PluginRequirement` so that if the other plugin changes this plugin will be able to tell and complain about it.